src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -227,6 +227,9 @@ pub const KATA_ANNO_CFG_HYPERVISOR_MEMORY_PREALLOC: &str =
 /// A sandbox annotation to specify if the memory should be pre-allocated from huge pages.
 pub const KATA_ANNO_CFG_HYPERVISOR_HUGE_PAGES: &str =
     "io.katacontainers.config.hypervisor.enable_hugepages";
+/// A sandbox annotation to specify huge page mode.
+pub const KATA_ANNO_CFG_HYPERVISOR_HUGE_PAGES_MODE: &str =
+    "io.katacontainers.config.hypervisor.enable_hugepages_mode";
```

Comment:
Thx.
What's about changing the boolean of `self.config.memory_info.enable_hugepages` to String ? 
And if so,  it'll avoid adding another config setting item for hugepage in configuration.

Reply:
Hi, I changed the self.config.memory_info.enable_hugepages to self.config.memory_info.hugepages of type string. Just change code part, If this change fine, I would modify the rest parts, like toml and docs.